tests/test_impersonated_credentials.py

```diff
@@ -62,10 +62,13 @@ class TestImpersonatedCredentials(object):
     SOURCE_CREDENTIALS = service_account.Credentials(
             SIGNER, SERVICE_ACCOUNT_EMAIL, TOKEN_URI)
 
-    def make_credentials(self, lifetime=LIFETIME):
+    def make_credentials(self, lifetime=LIFETIME, target_principal=None):
```

Comment:
Would it be possible to do `target_principal=self.TARGET_PRINCIPAL` and remove the `if target_principal is None ...` statement below?